### PR TITLE
Remove ping from tests to avoid failures on azp

### PR DESCRIPTION
**Pull request by Frawless**
Signed-off-by: Jakub Stejskal <xstejs24@gmail.com>

### Type of change

- Bugfix

### Description

This PR removes ping from several tests, because ping outside of the internal network is not allowed on azure pipelines infra.

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

**Review by scholzj — COMMENTED** [review]
So why are these not supported on AZP?

**Comment by Frawless** [comment]
@scholzj  ping from azp VMs is not allowed

**Comment by scholzj** [comment]
Since we want to focus the tests on AZP as our main infra (instead of the private Jenkins) ... would it make sense to use something else then ping? Or remove the ping and just check the `/etc/hosts` file?

**Comment by Frawless** [comment]
Yes checking /etc/hosts makes better sense. Going to rewrite it.

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=regression testcase=**/*ST#testHostAliases test_only

File: systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeST.java
Comment (scholzj) [inline systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeST.java:267]: Should these be in some abstract class or somewhere once instead of being in any file?

File: systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeST.java
Comment (scholzj) [inline systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeST.java:283]: Should the string you look for e defined next to the alias IP and hostname?

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=regression testcase=**/*ST#testHostAliases test_only

**Review by scholzj — APPROVED** [review]

File: systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMakerST.java
Comment (im-konge) [inline systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMakerST.java:742]: Sorry that I'm reviewing this, but can you please remove the `Matchers`? It's leftover from my PR...

**Review by im-konge — APPROVED** [review]

**Review by stanlyDoge — APPROVED** [review]

**Comment by scholzj** [comment]
@Frawless I know you are sick ... but is there any reason to not merge this?

**Comment by Frawless** [comment]
@scholzj there was one comment from Lukas about some old leftover in code which could be removed by this pr, but I think we can do it later. Sorry for delay, feel free to merge it.

**Comment by scholzj** [comment]
I will merge it once the CI passes so that we get the regression tests fixed. The leftover can be fixed in a separate PR. (cc @im-konge)

**Comment by im-konge** [comment]
@scholzj @Frawless merge it :) it was just an idea :) thanks

**Pull request merged**
